src/commands.rs

```diff
@@ -1,15 +1,199 @@
-use crate::Authentication;
+use crate::{Authentication, AuthenticationError};
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
 
+use serde_json::Value;
+
+#[derive(Error, Debug)]
+pub enum CommandError {
+    #[error("auth error")]
+    AuthenticationError(#[from] AuthenticationError),
+    #[error("request error")]
+    RequestError(#[from] reqwest::Error),
+    #[error("parse error")]
+    ParseError(#[from] serde_json::Error),
+    #[error("unknown error #[0]")]
+    WrongResponseStatus(u16),
+}
+// TODO: Use in the future no need to deserialize for current features
+#[derive(Debug, Deserialize, Serialize, Default)]
+pub struct ScreenConfig {}
+// TODO: Use in the future no need to deserialize for current features
+#[derive(Deserialize, Serialize, Debug, Default)]
+pub struct Screen {
```

Comment:
Is serialization here forward comptatible?
Meaning if any new field is added on the v3/v4 - will it just ignore it or fail?
If not sure - let's test it.